### PR TITLE
include information about creating a local build

### PR DESCRIPTION
**Pull request by kkovary**
A couple of steps were missing from `CONTRIBUTING.md`:

1. install poetry
2. build and install local package

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,6 +35,16 @@ Navigate into the repo:
 cd pynecone
 ```
 
+Install poetry:
+```
+pip install poetry
```
Comment (picklelo) [inline CONTRIBUTING.md:40]: I think this should be `curl -sSL https://install.python-poetry.org | python3 -`
Based on: https://python-poetry.org/docs/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,6 +35,16 @@ Navigate into the repo:
...
+```
+
+Install your local Pynecone build:
+```
+python -m pip install -e .
```
Comment (picklelo) [inline CONTRIBUTING.md:45]: Here we should run `poetry install` instead to install the Pynecone project requirements.

**Review by picklelo — COMMENTED** [review]
Thanks for catching this! Could you update the commands and I will approve this.

**Review by picklelo — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (picklelo) [reply CONTRIBUTING.md]: Great idea

**Comment by kkovary** [comment]
this is great!

**Pull request merged**